### DISCUSSION TITLE
Two accounts, the same IP address-is it a problem?

### DISCUSSION THREAD

**Discussion by Mareknown** [Chia-Network/chia-blockchain#3198] [Q&A]
We have two computers at home, one is connected to the internet provider router by ethernet cable and my wife's one is  connected by wireless dongle to it (so there is the same IP address).  We have two separate Chia accounts (GUI, two keys) and we plot independently (something like competition). The problem is that my old PC always synchronizes to Chia instantly but my wife's one, a brand new PC has always problem with the synchronization. I have to restart her computer and wait for an about hour for the synchronization. Another problem is with the Chia drop, we do not have a big chances to get the XCH but according to my calculation she should have at least 6 ...but she had just a single drop, a long time ago. Is it possible that there is a conflict between our computers since they are connected to the same IP address? We are using the newest Chia ver.1.1.2.

**Comment by pangchia** [comment]
i have same question

**Reply by Mareknown** [reply]
Of course, there is no answer, either the problem is too obvious or we are just too small to just bother with us. If you do not know programming or more pc stuff..get lost -:(   Yes, there are thousands of "small" people which invested in the project and wanted to know something more, for example, why the Chia is showing that everything is ok when you have no chance for winning a single chia!!! Reading comments, watching YouTube, I have came to conclusion that we have a problem indeed, having the same router IP we just cannot assign the famous 8444 port to both machines. Why there are no more ports available? Why the Chia GUI is showing that everything is ok...if it doesn't? Did you address the project just to people which know how to earn on it? Why did not inform us about it?  Just feel down with your approach....

**Comment by fisch88** [comment]
Anyone have an answer to this question?

**Reply by Mareknown** [reply]
It looks that they are working on the problem _https://chiaforum.com/t/alternative-to-port-8444-need-help-vpn/1348_
I was thinking about the solution described here for my pc:  "Establish a VPN tunnel through the NAT to a cloud server with a public IP address" _https://github.com/Chia-Network/chia-blockchain/wiki/Resolving-Sync-Issues---Port-8444_ but it is quite complicated for the beginners and I think I would not have an access to the Internet (or limited access) on my machine (I use the machine for other thing as well). 
For now, I just set up my virgin router to forward the 8444 to my wife PC and left the upnp ON. It works for her since she got a XCH last weekend. There are no drops on my machine but it looks like it is farming (dots are green). If I have a problem with synchronization on my machine, I just swap the 8444 port with my IP for about 10min and in that time the list of nodes populates to about 50, then I change the IP back to my wife one (you know how it works -)). My computer gradually looses the nodes so I repeat the process more or less every 12 hours (to maintained at least 20 nodes). On my wife machine everything goes smoothly while on mine I can see a huge nodes delay in response to the Header hashes (for example a height hash is 22000 while the nodes show 21960).....I just hope that it does not interfere with the XCH farming. 
The port checker application shows that on both machines the port 8444 for our IPs is opened.
